docs/contributors/index.md

```diff
@@ -41,12 +41,14 @@ By contributing, you agree that your contributions will be licensed as followed:
 
 ## Community
 
-Some of you might have noticed we have official community blog hosted on
-[Medium][k0s@medium]. If you are not yet following us, we'd like to invite you
-to do so now! Make sure to [follow us on X][@k0sproject] as well 😊
+Some of you may have noticed that we have a [community blog]. If you're not
+following us yet, we invite you to do so now! Make sure to [follow us on
```

Comment:
There's an RSS feed for the blog, right? We could add the link here. What is the link actually?